heartbeat/VirtualDomain

```diff
@@ -592,6 +634,14 @@ VirtualDomain_start() {
 	# is restored to an 'undefined' state before creating.
 	verify_undefined
 
+    if ocf_is_true "{OCF_RESKEY_start_resources}"; then
```

Comment:
Indentation for this section doesnt match the rest of the agent.

Reply:
oops! my bad I forgot to disable tab expansion again in my editor